components/ledger/internal/bootstrap/balance_sync.worker.go

```
@@ -0,0 +1,609 @@
+				"organization_id": organizationID.String(),
+				"ledger_id":       ledgerID.String(),
+				"mode":            "batch",
+			}).Add(ctx, result.BalancesSynced); metricErr != nil {
+				w.logger.Log(ctx, libLog.LevelWarn, "BalanceSyncWorker: failed to emit synced counter", libLog.Err(metricErr))
+			}
+		}
+	}
+
+	if result.KeysProcessed > 0 {
+		w.logger.Log(ctx, libLog.LevelInfo, "BalanceSyncWorker: batch sync completed",
+			libLog.Int("processed", int(result.KeysProcessed)),
```

golangci-lint (2.11.4)

[error] unnecessary conversion

(unconvert)